src/core/math/vec2/distance.test.js

```diff
@@ -0,0 +1,30 @@
+const test = require('ava')
+const {distance, fromValues} = require('./index')
+
+const {nearlyEqual} = require('../../../../test/helpers/index')
+const {EPS} = require('../../constants')
```

Comment:
are you sure about the use of EPS here ? we have not looked at that one in  along while

Reply:
I’ve been using EPS for all the nearlyEqual tests, thinking that this would be a better value then just something arbitrary. We can review EPS when working on the geometry / BSP trees, which is where EPS becomes important.